config-freebsd.md

```diff
@@ -0,0 +1,124 @@
+# <a name="FreeBSDContainerConfiguration" />FreeBSD Container Configuration
+
+This document describes the schema for the [FreeBSD-specific section](config.md#platform-specific-configuration) of the [container configuration](config.md).
+
+## <a name="configFreeBSDDevices" />Devices
+
+Devices in FreeBSD are accessed via the `devfs` filesystem. Typically, each container will have a `devfs` filesystem mounted into it's `/dev` directory. Often, a minimal set of devices is exposed to the container using ruleset 4 from `/etc/defaults/devfs.rules` - the ruleset is specified as a mount option.
+
+Optionally, additional devices can be exposed to the container using an array of entries inside the `devices` root field:
+
+* **`path`** _(string, REQUIRED)_ - the device path relative to `/dev`
+* **`mode`** _(integer, OPTIONAL)_ - device permissions as an integer which is interpreted as in `chmod(1)`.
+
+### Example
+```json
+"devices": [
+	{
+        "path": "pf",
+        "mode": "0700"
```

Comment:
Is it an integer or a string? 😅

Reply:
This is a string which is interpreted as in chmod(1) which supports octal numeric modes as well as symbolic. I will fix the text to make this a little clearer.

Reply:
This is a very good point. In this case, the parsing is handled by the devfs(8) system utility but that doesn't mean that this specification needs to allow anything more than a simple three digit octal number. I will rearrange things to share the definition of fileMode with config-linux.json and change this field to use that.